src/commands.ts

```diff
@@ -513,50 +513,71 @@ export const run = async (functionName: string, cmd: { nondev: boolean }) => {
 
 /**
  * Deploys an Apps Script project.
- * @param version {string} The project version to deploy at.
- * @param description {string} The deployment's description.
+ * @param cmd.versionNumber {string} The project version to deploy at.
+ * @param cmd.desc {string} The deployment description.
+ * @param cmd.deploymentId  {string} The deployment ID to redeploy.
  */
-export const deploy = async (version: number, description = '') => {
+export const deploy = async (cmd :{
+  versionNumber: number,
+  desc: string,
```

Comment:
`description = ''`,

Reply:
With Typescript, We can not be set Initial value inside type definition...